handlers/commands/report.js

```diff
@@ -35,6 +35,7 @@ const reportHandler = async ctx => {
 		reply_to_message_id: ctx.message.reply_to_message.message_id,
 	});
 	if (chats.report) {
+		await ctx.deleteMessage();
```

Comment:
No biggie, but this will lead to attemtping to delete the message twice sometimes

Reply:
>Bad, it'd be the only command to register monospaced. I'd add a check to do nothing in that case.

I've talked about this with Thomas and he said currently there's no checks being done if a command is monospace or not.

I wouldn't mind it ignoring monospace(to be fair I don't see people using monospace commands outside of jokes), but I have no idea how to do the check in the regex for that.

Reply:
Monospace can't be checked via regex, inspect `ctx.entities`.

> currently there's no checks being done if a command is monospace or not

False: https://github.com/thedevs-network/the-guard-bot/blob/7ea9f48e3fc54564a6701cd96f452e690b34ed60/handlers/commands/routingFn.js#L8 https://github.com/thedevs-network/the-guard-bot/blob/7ea9f48e3fc54564a6701cd96f452e690b34ed60/utils/cmd/parse.ts#L38-L46

Reply:
> currently there's no checks being done if a command is monospace or not

Well this is not what I claimed, I merely said that no checks are being done on whether `/report` is monospaced or not